src/Stratis.Bitcoin.Tests/Controllers/NodeControllerTest.cs

```diff
@@ -61,9 +61,9 @@ public NodeControllerTest()
         }
 
         [Fact]
-        public void Stop_WithFullNode_DisposesFullNodeAsync()
+        public async Task Stop_WithFullNode_DisposesFullNodeAsync()
         {
-            IActionResult result = this.controller.Shutdown();
+            IActionResult result = await this.controller.ShutdownAsync(0);
```

Comment:
In `NodeController` the method is still called `Shutdown()`.  

When I loaded this PR locally I get "NodeController does not contain a definition for nodeShutdownAsync...".

Reply:
Also does `ShutdownAsync()` need `.ConfigureAwait(false)` in tests ?

Reply:
Apologies, I might have screwed it up. Let me check.